final/MealsharingNet/IReviewRepository.cs

```diff
@@ -0,0 +1,12 @@
+using MealsharingNet.Models;
+
+namespace MealsharingNet;
+
+public interface IReviewRepository
+{
+    Task <IEnumerable<Review>> ListReviews();
+    Task AddReview(Review review);
+    Task<List<Review>> GetMealReviews(int mealId);
```

Comment:
For consistency,  the parameter here, `mealId`, should be named the same in the ReviewRepository and in the ReviewController. (The same for the Reservations) :)